docs/normalize-intervals.rst

```diff
@@ -0,0 +1,827 @@
+.. _normalize-intervals:
+
+====================================
+Normalize time series data intervals
+====================================
+
+.. sidebar:: The ISS Orbit
+
+    .. image:: _assets/img/normalize-intervals/orbit.png
+        :alt: An animated visualization of the ISS orbit
+        :target: https://en.wikipedia.org/wiki/International_Space_Station#/media/File:Animation_of_International_Space_Station_trajectory.gif
+
+    The ISS travels at 27,724 kilometers per hour and orbits Earth
+    approximately once every 90 minutes. Each complete wave on the graph
+    represents a full orbit. The amplitude oscillates
+
+If you followed one of the tutorials in the :ref:`previous section <gen-ts>`,
+you should have some mock time-series data about the position, or `ground
+point`_, of `International Space Station`_ (ISS).
+
+It is common to want to visualize time series data by graphing values over
+time. However, you may run into the following issues:
+
+1. The resolution of your data does not match the resolution you want for your
+   visualization.
+
+   *For example, you want to plot a single value per minute, but your data is
+   spaced in 10-second intervals. You will need to resample the data.*
+
+2. Your data is non-continuous, but you want to visualize a continuous time
+   series.
+
+   *For example, you want to plot every minute for the past 24 hours, but you
+   are missing data for some intervals. You will need to fill in the missing
+   values.*
+
+In this tutorial, we will demonstrate the shortcomings of visualizing the
+non-normalized data and show you how to address these shortcomings by
+normalizing your data using SQL.
+
+.. NOTE::
+
+    This tutorial focuses on the use of SQL. We have chosen to use a Python
+    client for demonstration purposes. However, *the following techniques will
+    work in any environment that allows for the execution of SQL*.
```

Comment:
remove the first person?

Reply:
I can't figure out how to do that without sacrificing the active voice :(

Reply:
How about:

"In this tutorial, you will see the shortcomings of visualizing the non-normalized data and how to address these shortcomings by normalizing your data using SQL."

"You will use the Python client for demonstration purposes."

Reply:
went with this:

> This tutorial focuses on the use of SQL. Code examples use the CrateDB Python client for demonstration purposes. However, *the following techniques will work with any language that allows for the execution of SQL*.